app/services/match_orders.rb

```diff
@@ -30,7 +30,7 @@ def match_orders(sell_orders, buy_orders)
   private
 
   def orders_match?(buy_order, sell_order)
-    !sell_order.fulfilled &&
+    sell_order.fulfilled_at.nil? &&
```

Comment:
Mmmm that would make my code read nicer, sounds good to me.

On Tue, 27 Oct 2015 at 18:25 Nick Johnstone <notifications@github.com>
wrote:

> In app/services/match_orders.rb
> <https://github.com/akampjes/devtrain-stockexchange/pull/3#discussion_r43085314>
> :
>
> > @@ -30,7 +30,7 @@ def match_orders(sell_orders, buy_orders)
> >    private
> >
> >    def orders_match?(buy_order, sell_order)
> > -    !sell_order.fulfilled &&
> > +    sell_order.fulfilled_at.nil? &&
>
> Maybe you could have a fufilled? method that does fufilled_at.present?
>
> Would solve your test problem too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akampjes/devtrain-stockexchange/pull/3/files#r43085314>
> .
>